usr/src/uts/common/fs/zfs/arc.c

```diff
@@ -3424,8 +3424,8 @@ arc_reclaim_thread(void)
 			 * even if we aren't being signalled)
 			 */
 			CALLB_CPR_SAFE_BEGIN(&cpr);
-			(void) cv_timedwait(&arc_reclaim_thread_cv,
-			    &arc_reclaim_lock, ddi_get_lbolt() + hz);
+			(void) cv_timedwait_hires(&arc_reclaim_thread_cv,
+			    &arc_reclaim_lock, SEC2NSEC(1), MSEC2NSEC(1), 0);
```

Comment:
> On Feb 24, 2016, at 7:03 AM, Robert Mustacchi <notifications@github.com> wrote:
> 
> In usr/src/uts/common/fs/zfs/arc.c <https://github.com/openzfs/openzfs/pull/70#discussion_r53950489>:
> 
> > @@ -3424,8 +3424,8 @@ arc_reclaim_thread(void)
> >  			 * even if we aren't being signalled)
> >  			 */
> >  			CALLB_CPR_SAFE_BEGIN(&cpr);
> > -			(void) cv_timedwait(&arc_reclaim_thread_cv,
> > -			    &arc_reclaim_lock, ddi_get_lbolt() + hz);
> > +			(void) cv_timedwait_hires(&arc_reclaim_thread_cv,
> > +			    &arc_reclaim_lock, SEC2NSEC(1), MSEC2NSEC(1), 0);
> For what it's worth, we've been running everything with hires ticks and haven't noticed any changes here or issues from more aggressive arc reclaim activity.
> 
> 

ditto, I've been running hires for a few years now and do monitor arc reclaim activity continuously.
Over in the Solaris camp, they are tickless, but still suffer from reclaim issues through 11.1.
 -- richard